src/main/java/appeng/fluids/container/slots/IMEFluidSlot.java

```diff
@@ -27,7 +27,12 @@
  * @version rv6
  * @since rv6
  */
-public interface IMEFluidSlot extends IFluidSlot
+public interface IMEFluidSlot
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Annotate the "IMEFluidSlot" interface with the @FunctionalInterface annotation [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1609)